postgresql/values.yml

```diff
@@ -6,3 +6,5 @@ auth:
 primary:
   persistence:
     size: 2Gi
+  service:
+    type: LoadBalancer
```

Comment:
This will tell kubernetes to create an external-facing load balancer for the DB service. Your cloud provider will create the load balancer.